changelog/814.fix.rst

```diff
@@ -0,0 +1 @@
+Use cache for threads in interaction data instead of always creating a new instance.
```

Comment:
this is an invalid category, should be named `changelog/814.bugfix.rst`

Reply:
oops, good catch
https://github.com/DisnakeDev/disnake/pull/814/commits/d40eb6cd71e8c1a6cd410472b04e6fc452b79fe0